dev-cpp/waylandpp/waylandpp-0.1.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit multilib scons-utils toolchain-funcs versionator
+
+DESCRIPTION="Wayland C++ bindings"
+HOMEPAGE="https://github.com/NilsBrause/waylandpp"
+
+LICENSE="MIT"
+IUSE="doc"
+SLOT="0/$(get_major_version)"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/NilsBrause/waylandpp.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/NilsBrause/waylandpp/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/waylandpp-${PV}"
+fi
+
+COMMON_DEPEND="
+	>=dev-libs/wayland-1.11.0
+	media-libs/mesa[wayland]
+"
+DEPEND="${COMMON_DEPEND}
+	doc? (
+		app-doc/doxygen
+		media-gfx/graphviz
+	)
+	"
+RDEPEND="${COMMON_DEPEND}"
```

Comment:
this is one of those cases where imo `COMMON_DEPEND` is unnecessary: put `COMMON_DEPEND` in `RDEPEND`, and include `RDEPEND` in `DEPEND`. Less variables, easier